lib/s3routes/routesUtils.ts

```diff
@@ -495,6 +493,7 @@ function _computeContentLengthFromLocation(
                 return sum + parseInt(location.size, 10);
             }
         }
+        return sum;
```

Comment:
As sum is undefined here,

```suggestion
        return;
```